spec/controllers/recipe_revisions_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe RecipeRevisionsController do
+  let!(:user) { recipe.user }
```

Comment:
As spec files get bigger I generally find that the use of `let` makes them harder to read / understand. Though that's more of a personal style thing.

How do other people feel? /cc @dpbus